src/main/java/rx/Observable.java

```diff
@@ -11439,6 +11439,49 @@ public final Subscription subscribe(Subscriber<? super T> subscriber) {
     }
 
     /**
+     * Returns an Observable that emits the events emitted by source Observable, in a
+     * sorted order. Each item emitted by the Observable must implement {@link Comparable} with respect to all
+     * other items in the sequence.
+     * <dl>
```

Comment:
Please add a note about long or non-terminating or infinite sources as they may run out of memory or never finish collecting the elements to be sorted.